ivy_tests/test_ivy/test_frontends/test_tensorflow/test_nn.py

```diff
@@ -1340,7 +1340,7 @@ def test_tensorflow_softmax(
 @handle_frontend_test(
     fn_tree="tensorflow.nn.embedding_lookup",
     dtypes_indices_weights=helpers.embedding_helper(),
-    max_norm=st.floats(min_value=0, max_value=5, exclude_min=True),
+    max_norm=st.floats(min_value=0.1, max_value=5, exclude_min=True),
```

Comment:
Values smaller than 0.1 still work fine, but there is slight imprecision when working with very small values which can cause the test to fail.